plugins/obs-ffmpeg/data/locale/en-US.ini

```diff
@@ -49,6 +49,9 @@ Play="Play"
 Pause="Pause"
 Stop="Stop"
 
+FFmpeg.Directives="FFmpeg Directives"
+FFmpeg.Directives.Tooltip="Allows you to provide extra parameters to FFmpeg. This only accepts simple parameters in a key=value format, eg rtsp_transport=tcp\nMultiple parameters can be passed provided they are separated by a space, eg rtsp_transport=tcp rtsp_flags=prefer_tcp\n"
```

Comment:
"eg" is always written as "e.g." and is either used in a parenthetical or an appositive phrase.

> This only accepts simple parameters in a key=value format, e.g., rtsp_transport=tcp.\n
> This only accepts simple parameters in a key=value format (e.g., rtsp_transport=tcp).\n

Alternatively, a colon would work as well.

> This only accepts simple parameters in a key=value format:  rtsp_transport=tcp.\n

Or use a phrase such as "such as" to connect the two parts:

> This only accepts simple parameters in a key=value format, such as rtsp_transport=tcp.\n

Reply:
Would it be acceptable to break each examples out onto a separate line?

```
This only accepts simple parameters in a key=value format.
Example: rtsp_transport=tcp
```

Reply:
If you want them on their own line, that's fine.  Though, if you do that, then I think it's fine to just use the one example at the end for `rtsp_transport=tcp rtsp_flags=prefer_tcp`.  I don't know that there's any additional value having one example showing how the `key=value` syntax works and a separate example showing how to separate them by spaces.  Maybe someone else has a different opinion or better reasoning to provide here.

Reply:
Is this acceptable?  I want to get this right for a squash

FFmpeg.Options="FFmpeg Options"
FFmpeg.Options.Tooltip="Allows you to provide extra parameters to FFmpeg. This only accepts parameters in the option=value format.\nMultiple parameters can be passed provided they are separated by a space.\nExample: rtsp_transport=tcp rtsp_flags=prefer_tcp"

Reply:
Please use "options" instead of "parameters" in the tooltip text, per my other review comment.  You can use simpler sentences for this:

> FFmpeg.Options.Tooltip="Allows you to set FFmpeg options. This only accepts options in the option=value format.\nMultiple options can be set by separating them with a space.\nExample: rtsp_transport=tcp rtsp_flags=prefer_tcp"

Reply:
i've copy/pasted into the new rewind/squash PR